### PR TITLE
[BXMSDOC-4566] Misha's editorial changes for 7.5 - 7.5.x branch

### PR DESCRIPTION
**Pull request by mramendi**
Signed-off-by: Mikhail Ramendik <mramendi@redhat.com>

Please merge, this is the 7.5.x cherry-pick

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [ ] Pull Request title is properly formatted: `[RHPAM-XYZ] Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Attached commits represent units of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [ ] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

File: templates/rhpam75-kieserver-externaldb.yaml [hunk 1]
```diff
@@ -32,7 +32,7 @@ parameters:
   value: myapp
   required: true
 - displayName: Maven mirror URL
-  description: Maven mirror that the KIE server must use. If you configure a mirror, this mirror must contain all artifacts that are required for building and deploying your services.
+  description: Maven mirror that the KIE server must use. If you configure a mirror, this mirror must contain all artifacts that are required for deploying and running your services.
```
Comment (spolti) [inline templates/rhpam75-kieserver-externaldb.yaml]: this changes sounds not ok for me.

File: templates/rhpam75-prod-immutable-kieserver-amq.yaml [hunk 2]
```diff
@@ -48,14 +48,14 @@ parameters:
   value: executionUser
   required: false
 - displayName: KIE Server Password
-  description: KIE server password, used to connect to KIE servers. Generated value can be a suggestion to use for thew s2i various (Sets the org.kie.server.pwd system property)
+  description: KIE server password. If this parameter is not set, the password is generated automatically. (Sets the org.kie.server.pwd system property)
```
Comment (spolti) [inline templates/rhpam75-prod-immutable-kieserver-amq.yaml]: I believe that "the password is automatically generated" sounds better.

File: templates/rhpam75-prod-immutable-kieserver-amq.yaml [hunk 3]
```diff
@@ -165,12 +165,12 @@ parameters:
...
   required: false
 - displayName: KIE Server Bypass Auth User
-  description: KIE server bypass auth user (Sets the org.kie.server.bypass.auth.user system property)
+  description: Allows the KIE server to bypass the authenticated user for task-related operations, for example, queries. (Sets the org.kie.server.bypass.auth.user system property)
   name: KIE_SERVER_BYPASS_AUTH_USER
   value: 'false'
   required: false
 - displayName: KIE Server Container Deployment
-  description: 'KIE Server Container deployment configuration in format: containerId=groupId:artifactId:version|c2=g2:a2:v2'
+  description: 'KIE Server Container deployment configuration with optional alias. Format: containerId=groupId:artifactId:version|c2(alias2)=g2:a2:v2'
```
Comment (spolti) [inline templates/rhpam75-prod-immutable-kieserver-amq.yaml]: Will let the review of this field to @errantepiphany.

File: templates/docs/rhpam75-trial-ephemeral.adoc [hunk 4]
```diff
@@ -310,7 +310,7 @@ Http Get on http://localhost:8080/services/rest/server/healthcheck
 |`KIE_MBEANS` | KIE server mbeans enabled/disabled. (Sets the kie.mbeans and kie.scanner.mbeans system properties) | `${KIE_MBEANS}`
 |`DROOLS_SERVER_FILTER_CLASSES` | KIE server class filtering. (Sets the org.drools.server.filter.classes system property) | `${DROOLS_SERVER_FILTER_CLASSES}`
 |`PROMETHEUS_SERVER_EXT_DISABLED` | If set to false, the prometheus server extension will be enabled. (Sets the org.kie.prometheus.server.ext.disabled system property) | `${PROMETHEUS_SERVER_EXT_DISABLED}`
-|`KIE_SERVER_BYPASS_AUTH_USER` | Allows the KIE server to bypass the authenticated user for task related operations e.g. queries. (Sets the org.kie.server.bypass.auth.user system property) | `${KIE_SERVER_BYPASS_AUTH_USER}`
+|`KIE_SERVER_BYPASS_AUTH_USER` | Allows the KIE server to bypass the authenticated user for task-related operations e.g. queries. (Sets the org.kie.server.bypass.auth.user system property) | `${KIE_SERVER_BYPASS_AUTH_USER}`
```
Comment (spolti) [inline templates/docs/rhpam75-trial-ephemeral.adoc]: adoc files are auto generated, changes made here will be lost.

File: templates/rhpam75-kieserver-externaldb.yaml (hunk 1 above)
Comment (mramendi) [reply templates/rhpam75-kieserver-externaldb.yaml]: How? This template does not build anything.

File: templates/docs/rhpam75-trial-ephemeral.adoc (hunk 4 above)
Comment (mramendi) [reply templates/docs/rhpam75-trial-ephemeral.adoc]: I will remove adoc changes once we work out everything else and I re-push

File: templates/rhpam75-prod-immutable-kieserver-amq.yaml (hunk 2 above)
Comment (mramendi) [reply templates/rhpam75-prod-immutable-kieserver-amq.yaml]: While it does not seem so to me, I can make this change.

File: templates/rhpam75-prod-immutable-kieserver-amq.yaml (hunk 3 above)
Comment (mramendi) [reply templates/rhpam75-prod-immutable-kieserver-amq.yaml]: I got this change from @errantepiphany and he reviewed it already in the master branch - this is a cherry-pick

File: templates/rhpam75-kieserver-externaldb.yaml (hunk 1 above)
Comment (spolti) [reply templates/rhpam75-kieserver-externaldb.yaml]: "for deploying your...." sounds enough to me,.

File: templates/rhpam75-prod-immutable-kieserver-amq.yaml (hunk 3 above)
Comment (spolti) [reply templates/rhpam75-prod-immutable-kieserver-amq.yaml]: ok, no problem.

**Pull request merged**